Jenkinsfile

```diff
@@ -215,9 +235,29 @@ node {
 
   stage ("run akamai production smoke tests") {
     try {
-      openShiftUtils.withNode {
-        sh "iqe plugin install akamai"
-        sh "IQE_AKAMAI_CERTIFI=true DYNACONF_AKAMAI=\'@json {\"release\":\"${RELEASESTR}\"}\' iqe tests plugin akamai -s -m ${PRODTESTSTR}"
+      if (RUN_SMOKE_TESTS) {
+        openShiftUtils.withNode {
+          withCredentials([
+            string(credentialsId: "vaultRoleId", variable: 'DYNACONF_IQE_VAULT_ROLE_ID'),
+            string(credentialsId: "vaultSecretId", variable: 'DYNACONF_IQE_VAULT_SECRET_ID'),
+          ]) {
+            // set some env variables for authentication in the iqe-core pod
+            withEnv([
+              "DYNACONF_IQE_VAULT_ROLE_ID=$DYNACONF_IQE_VAULT_ROLE_ID",
+              "DYNACONF_IQE_VAULT_SECRET_ID=$DYNACONF_IQE_VAULT_SECRET_ID",
+              "DYNACONF_IQE_VAULT_LOADER_ENABLED=true",
+              "ENV_FOR_DYNACONF=prod"
+            ]) {
+              // install akamai and 3scale plugins, run smoke tests
+              sh "iqe plugin install akamai 3scale"
+              sh "IQE_AKAMAI_CERTIFI=true DYNACONF_AKAMAI=\'@json {\"release\":\"${RELEASESTR}\"}\' iqe tests plugin akamai -s -m ${PRODTESTSTR}"
+              sh "iqe tests plugin 3scale --akamai-production -m akamai_smoke"
```

Comment:
@john-dupuy just to confirm, will this run against cloud stage (cloud.stage.redhat.com) or against the staging version of the production Akamai config (cloud.redhat.com) before it's promoted to the Akamai production config?

Reply:
@coderbydesign this one specifically will run against the Akamai production config (`--akamai-production`)

https://github.com/RedHatInsights/cloud-services-config/pull/746/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R134 is where they will run against the staging version of the prod akamai config (`--akamai-staging`)

Unfortunately we cannot run these tests against the stage env `cloud.stage.redhat.com` because the proxy requires a VPN connection and these tests run outside of VPN.

Reply:
@john-dupuy got it, ok. Thanks for clarifying! So that will at least gate the staging config before it moves to prod.

Reply:
Yup, precisely!